et_ee.lang

```diff
@@ -406,4 +406,13 @@ gui.xaero_ui_scale=Liidese suurus
 gui.xaero_ui_scale_auto=autom.
 gui.xaero_ui_scale_mc=Minecraft
 gui.xaero_box_ui_scale=Teksti ja teiste minikaardi liidese elementide suurus. Pole mõeldud suumimiseks ega suuruse muutmiseks, selleks on teised valikud.
-gui.xaero_no_durability=puudub 
+gui.xaero_no_durability=puudub
```

Comment:
This should be capitalized and I don't think it is correct. It's one of the options for displaying armor durability that users can purposely choose.

Reply:
Well, it is the word that "none" commonly translates to, but I could also use "hidden" if that's better?

Reply:
Then I think it should work, so no need to change it.